application/controllers/users_controller.py

```diff
@@ -0,0 +1,91 @@
+"""
+Handles all logic of the user api
+"""
+from flask import jsonify, make_response
+from neomodel import exceptions
+from flask_jwt_extended import (jwt_required, create_access_token, unset_jwt_cookies,
+                                set_access_cookies, get_csrf_token)
+from webargs.flaskparser import use_args
+
+from .application_controller import ApplicationController
+from ..models.user import User
+from ..responders import respond_with, no_content
+from ..validators import users_validator
+from ..extensions import bcrypt
+
+class UsersController(ApplicationController):
+    """ Controller for users """
+
+    @jwt_required
```

Comment:
okay so I checked the docs. This checks if a jwt token is present. But does it check if it belongs to the user that is trying to be accessed? How do we configure pages and resources to only be avaiable for one person. So that for example a user can only see his own preferences page but not that of others?

Reply:
the user_id would be decoded in the jwt token for permission control